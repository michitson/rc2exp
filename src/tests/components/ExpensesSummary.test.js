import React from 'react'
import {shallow} from 'enzyme'
import {ExpensesSummary} from '../../components/ExpensesSummary'

beforeEach(() =>{

})

test('should correctly render expense summary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={121212}/>)
  expect(wrapper).toMatchSnapshot()
}) 

test('should correctly render expense summary with > 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={13} expensesTotal={34343434343}/>)
  expect(wrapper).toMatchSnapshot()
}) 

test('', () => {
  
}) 

test('', () => {
  
}) 

test('', () => {
  
}) 

test('', () => {
  
}) 