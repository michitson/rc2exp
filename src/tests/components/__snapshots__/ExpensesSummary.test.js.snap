// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expense summary with > 1 expense 1`] = `
<div>
  <h1>
    Viewing 13 expenses totalling $343,434,343.43
  </h1>
</div>
`;

exports[`should correctly render expense summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 1 expense totalling $1,212.12
  </h1>
</div>
`;
